kuma/static/js/wiki.js

```diff
@@ -191,11 +191,21 @@
         Syntax highlighting scripts
     */
     if($('article pre').length && ('querySelectorAll' in doc)) (function() {
-        var syntaxScript = doc.createElement('script');
-        syntaxScript.setAttribute('data-manual', '');
-        syntaxScript.async = 'true';
-        syntaxScript.src = mdn.staticPath + 'js/syntax-prism-min.js?build=' + mdn.build;
-        doc.body.appendChild(syntaxScript);
+        var bundle = 'syntax-prism';
+        if (window.mdn && mdn.assets.js && mdn.assets.js.hasOwnProperty(bundle)) {
```

Comment:
I'm iterating over the list returned from ``mdn.assets.js['syntax-prism']``, after ``mdn.assets.js.hasOwnProperty('syntax-prism')`` checks that the key exists.  That check doesn't make sense inside the loop.